samples/sampler/stories/qa/FixedPopupPanels.js

```diff
@@ -242,3 +256,183 @@ WithScroller.parameters = {
 		text: 'QA -  Scroller with text inside FixedPopupPanels'
 	}
 };
+
+export const WithVariousItems = () => {
+	const defaultOpen = true;
+	const [open, setOpenState] = useState(defaultOpen);
+	const toggleOpen = () => setOpenState(!open);
+	const handleClose = compose(toggleOpen, action('onClose'));
+
+	const defaultIndex = 0;
+	const [index, setPanelIndexState] = useState(defaultIndex);
+
+	const nextPanel = () => setPanelIndexState(Math.min(index + 1, 3));
+	const prevPanel = () => setPanelIndexState(Math.max(index - 1, 0));
+	const handleBack = compose(prevPanel, action('onBack'));
+
+	// Navigate menus with the right key. The left key is handled by framework.
+	const handleKeyDown = (ev) => {
+		const {keyCode} = ev;
+
+		if (isRight(keyCode) && ev.target && !ev.target.hasAttribute('disabled')) {
+			nextPanel();
+		}
+	};
+
+	return (
+		<div>
+			<FixedPopupPanels
+				index={index}
+				open={open}
+				position={select('position', ['left', 'right'], Config)}
+				fullHeight={boolean('fullHeight', Config)}
+				width={select('width', ['narrow', 'half'], Config)}
+				noAnimation={boolean('noAnimation', Config)}
+				noAutoDismiss={boolean('noAutoDismiss', Config)}
+				onBack={handleBack}
+				onClose={handleClose}
+				onHide={action('onHide')}
+				onShow={action('onShow')}
+				scrimType={select('scrimType', ['none', 'translucent', 'transparent'], Config)}
+				spotlightRestrict={select('spotlightRestrict', ['self-first', 'self-only'], Config)}
+			>
+				<Panel>
+					<Header>
+						<title>Panel 1</title>
+						<subtitle>This is the subtitle</subtitle>
+						<slotAfter>
+							<Button size="small" icon="arrowlargeright" onClick={nextPanel} />
+						</slotAfter>
+					</Header>
+					<Column>
+						<Cell shrink component={BodyText}>
+							A 3-Cell Layout with various items
+						</Cell>
+						<Cell>
+							<span>This is a text.</span>
+							<Button size="small" disabled onClick={nextPanel} onKeyDown={handleKeyDown}>Button1</Button>
+							<br />
+							<br />
+							<Button size="small">Button2</Button>
+							<Button size="small" onClick={nextPanel} onKeyDown={handleKeyDown}>Button3</Button>
+							<br />
+							<br />
+							<Button size="small" disabled>Button4</Button>
+							<Dropdown width={100} style={{margin: 0}} title="A dropdown">
+								{['a', 'b', 'c', 'd', 'e', 'f']}
+							</Dropdown>
+							<br />
+							<Slider />
```

Comment:
How about store this L311~326 to a variable to reuse and avoid code duplication.